packages/pn-pa-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -67,6 +67,7 @@ describe('NotificationDetail Page (one recipient)', () => {
   let result: RenderResult;
   const mockDispatchFn = jest.fn();
   const mockActionFn = jest.fn();
+  const openModal = jest.fn();
```

Comment:
open the modal simulating the click and after check that the modal is in the document